src/averageAges.js

```diff
@@ -37,7 +38,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = (!withChildren)
```

Comment:
`const women = !withChildren`

Reply:
the habit of writing conditions in parentheses